package.json

```diff
@@ -23,8 +23,7 @@
 		"----- CI ---- used to test the codebase on every commit": "",
 		"ci": "pnpm install && pnpm check-all",
 		"check-all": "pnpm audit --audit-level=high && pnpm format && pnpm lint && pnpm build && pnpm test",
-		"ci:publish": "pnpm --filter !vs-code-extension publish -r",
-		"ci:version": "pnpm exec changeset version && pnpm install --lockfile-only"
+		"ci:publish": "pnpm --filter !vs-code-extension publish -r"
```

Comment:
why is the ide extension excluded?

Reply:
Not sure tbh. @felixhaeberle do you know why? Couldn't the Extension be marked as private if it shouldn't be published on NPM?